packages/ui/src/AthleteAvatarMenu/AthleteAvatarMenu.tsx

```diff
@@ -50,12 +50,12 @@ const AthleteAvatarMenu: React.FC<AthleteAvatarMenuProps> = ({
       <div className="mt-4">
         <p className="text-xs text-gray-600 uppercase">Actions</p>
         <div className="pl-3 pt-2">
-          {/* <button onClick={onAddAccount} className={itemClasses}>
+          <button onClick={onAddAccount} className={itemClasses}>
             <span className="align-middle inline-block h-5 w-5 mr-1">
               <Plus />
             </span>
             <span className="align-middle">Add account</span>
```

Comment:
`Add account`, `Other accounts` and `actions` need to be translated. I propose to remove `Other accounts` and `Actions`, and replace the latter with a line separator.

Reply:
Here, which one do you prefer:

The old one:
<img width="451" alt="Screenshot 2024-01-26 at 07 54 47" src="https://github.com/eisbuk/EisBuk/assets/72689023/da0fad6f-f51f-4b03-98c1-ef064b96e996">

The suggested update:
<img width="451" alt="Screenshot 2024-01-26 at 08 00 10" src="https://github.com/eisbuk/EisBuk/assets/72689023/0067cb69-9df7-469a-8d73-9b9bbb1badf3">

I favour the old one slightly, but I think both are good...

Reply:
The new one does look nicer, but the old one is more telling...

Reply:
I like the new one: there are also less strings to translate (and I don't dislike its appereance).